dandiapi/api/management/commands/refresh_metadata.py

```diff
@@ -0,0 +1,19 @@
+import djclick as click
+
+from dandiapi.api.models import Asset, Version
+
+
+@click.command()
+def refresh_metadata():
+    """
+    Re-save all Versions and Assets to refresh their metadata.
+
+    This script should be run after modifying the computed metadata for either Versions or Assets.
+    Since the computed metadata is only computed when the model is saved, the models need to be
+    saved to trigger computation.
+    """
```

Comment:
I wonder what are exact "effects" of running this command since it talks in the terms of "model is saved" etc.  E.g. -- would it mint new assets?  would "published" (released) assets be reminted?

Reply:
It does not mint new assets, but it might update the metadata of assets which have been published.